File: features/digital/event/src/main/java/com/tokopedia/events/data/entity/response/verifyresponse/Message.java
package com.tokopedia.events.data.entity.response.verifyresponse;

import com.google.gson.annotations.SerializedName;

public class Message{

	@SerializedName("title")
	private String title;

	@SerializedName("message")
	private String message;

	public void setTitle(String title){
		this.title = title;
	}

	public String getTitle(){
		return title;
	}

	public void setMessage(String message){
		this.message = message;
	}

	public String getMessage(){
		return message;
	}

	@Override
 	public String toString(){
		return 
			"Message{" + 
			"title = '" + title + '\'' + 
			",message = '" + message + '\'' + 
			"}";
		}
}

File: features/transaction/purchase_platform/src/main/java/com/tokopedia/purchase_platform/features/checkout/data/model/response/checkout/Message.kt
package com.tokopedia.purchase_platform.features.checkout.data.model.response.checkout

import com.google.gson.annotations.SerializedName

/**
 * Created by Irfan Khoirul on 2019-11-12.
 */

data class Message(
        @SerializedName("title")
        val title: String = "",

        @SerializedName("desc")
        val desc: String = "",

        @SerializedName("action")
        val action: String
)